src/Package/Impl/DataInspect/VariableView.xaml.cs

```diff
@@ -219,5 +273,85 @@ public partial class VariableView : UserControl, IDisposable {
                     return KnownMonikers.BulletList;
             }
         }
+        #endregion
+
+        #region ICommandTarget
+        public CommandStatus Status(Guid group, int id) {
```

Comment:
I don't understand why do you need this.
VE is a tool window, which should handle VS commands.